src/renderer.c

```diff
@@ -98,8 +99,9 @@ static int font_set_render_options(RenFont* font) {
   return 0;
 }
 
-static int font_set_style(FT_Outline* outline, int x_translation, unsigned char style) {
+static int font_set_style(FT_Outline* outline, int x_translation, unsigned char style, bool smoothing) {
   FT_Outline_Translate(outline, x_translation, 0 );
+  if(smoothing) FT_Outline_Embolden(outline, 1 << 5);
```

Comment:
5 might be too much, maybe 3 is a better value?

Reply:
I sort of just copied the thing they did in the original discussion, as that seemed to produce nice results. I don't know exactly what settings to do, so honestly I'll have to do a comparison somehow. We could also make the option a number if that's what's needed.